### PR TITLE
Home work 7 task 3 completed, check please

### PR DESCRIPTION
**Pull request by vladis1350**

### CODE REVIEW

File: homework/Vladislav Urbanski/NumberService/src/com/vladis1350/Application.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
+package com.vladis1350;
+
+import com.vladis1350.service.NumberService;
+
+public class Application {
+    public static void main(String[] args) throws InterruptedException {
+        NumberService numberServiceOne = new NumberService(66, 15);
+        NumberService numberServiceTwo = new NumberService(15, 2);
+        Thread thread = new Thread(numberServiceOne);
```
Comment (javaGuruBY) [inline homework/Vladislav Urbanski/NumberService/src/com/vladis1350/Application.java:9]: It would be better to give meaningful names to thread's variables

**Review by javaGuruBY — APPROVED** [review]
lgfm

**Review by akupreychik — APPROVED** [review]
lgfm

**Review by javaGuruBY — APPROVED** [review]
startMoreFinish ROFL

**Pull request merged**